scripts/create_tables.sql

```diff
@@ -1 +1,28 @@
 -- create doctors, patients, and appointments tables
+create table doctors(
+  id serial primary key,
+  surname varchar,
+  given_name varchar,
+  gender varchar(1),
+  dob date,
```

Comment:
I don't think that DATE will accept a string in the format found in the dob column of the .csv file. VARCHAR or TEXT would work.